src/console/commands.rs

```diff
@@ -0,0 +1,93 @@
+use bevy::prelude::*;
+use sysinfo::{ProcessorExt, System, SystemExt, UserExt};
+
+use super::{ConsoleData, event::{EnteredConsoleCommandEvent, PrintConsoleEvent}};
+
+pub fn commands_handler(
+    mut cmd_reader: EventReader<EnteredConsoleCommandEvent>,
+    mut console_writer: EventWriter<PrintConsoleEvent>,
+    mut data: ResMut<ConsoleData>,
+    mut sys: ResMut<System>,
+) {
+    for EnteredConsoleCommandEvent(cmd) in cmd_reader.iter() {
+        // Don't do anything if the string is empty
+        if cmd.is_empty() { return ; }
+
+        let args: Vec<&str> = cmd.trim().split(' ').collect();
+
+        if args[0] != "clear" {
+            // first send what the user typed
+            let mut user_input = String::from("> ");
+            user_input.push_str(cmd.clone().trim());
+            console_writer.send(PrintConsoleEvent(user_input));
+        }
+
+        match args[0] {
+            "clear" => data.messages.clear(),
+            "help" => console_writer.send(PrintConsoleEvent(display_help())),
+            "motd" => console_writer.send(PrintConsoleEvent(print_motd(&mut sys, true))),
+
+            _ => {
+                console_writer.send(PrintConsoleEvent(format!("I didn't understand the command: \"{}\"", args[0])));
+            }
+        }
+    }
+}
+
+fn display_help() -> String {
+    let mut res = String::from("\nSHOWING AVAILABLE COMMANDS\n");
+
+    let underline  = "==========================\n\n";
+    res.push_str(underline);
+
+    res.push_str("- help : Displays this message\n");
+    res.push_str("- clear : Clears commands on the screen\n");
+    res.push_str("- motd : Prints informations about YOUR computer\n");
+
+    res
+}
+
+pub fn print_motd(sys: &mut System, should_refresh: bool) -> String {
+    if should_refresh {
+        sys.refresh_cpu();
+        sys.refresh_memory();
+        sys.refresh_system();
+        sys.refresh_users_list();
+    }
+    
+    let mut res = String::from("Welcome back on SafeOS 3.1\n");
+    res.push_str("--------------------------\n");
+
+    res.push_str(&format!("Username: {}\n\n", sys.users().last().unwrap().name()));
+
+    res.push_str(&format!("System name:             {:?}\n", sys.name().unwrap()));
```

Comment:
Consider if we can have some kinda placeholder for these values instead of unwrap

Reply:
Wilco